Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -193,7 +198,12 @@ class PlaceCarouselViewController: UIViewController {
         // apply the constraints
         NSLayoutConstraint.activate(constraints, translatesAutoresizingMaskIntoConstraints: false)
 
-        toggleLoadingUI(loading: true)
+        toggleLoadingUI(loading: shouldFetchPlaces)
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        loadingOverlay.isHidden = false
```

Comment:
IIRC the issue with the non-touchable view was because of the scrolling needed during the transition/cell not found. Do we need this? Might be safer to exclude it if it's not required.